yoyodyne/evaluators.py

```diff
@@ -80,10 +77,75 @@ def evaluate(
         _, predictions = torch.max(predictions, dim=2)
         # Finalizes the predictions.
         predictions = self.finalize_predictions(predictions, end_idx, pad_idx)
+        golds = self.finalize_golds(golds, end_idx, pad_idx)
         return self.get_eval_item(predictions, golds, pad_idx)
 
-    @staticmethod
     def get_eval_item(
+        self,
+        predictions: torch.Tensor,
+        golds: torch.Tensor,
+        pad_idx: int,
+    ) -> EvalItem:
+        """Converts finalized predictions and golds into the metric.
```

Comment:
I don't know what people normally do here but I think you could safely omit the docs on the ABC? Any thoughts?

Reply:
Yeah I actually have no idea either. I think ill remove it.